### DISCUSSION TITLE
「开源之史」系列之二/二：真实世界中看到的开源和看不见的价值 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#40] [Announcements]
# 「开源之史」系列之二/二：真实世界中看到的开源和看不见的价值 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/002-what-is-seen-and-unseen-in-real-world/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

其实吧, 财务核算根本不是最关键的, 关键在: 为什么要看到?

其实, 人类就是这么狂妄自大的闯过来的, 在不知道 666 负作用前, 就当成完全安全的药剂成吨成吨的向城市/田野播撒....直到发现对遗传能产生不可逆损害时, 已经有多少生物遭到迫害?...

关键在, 好奇心被学校/媒体/公知/...大规模清除了;
要知道手机刚开始, 还有厂商宣传智能手机信号太强, 远没有功能机安全, 
常用会诱发脑癌....

如果我们认为企业专业财务不知道如何分析开源包含的隐藏成本? 那可能是我们自己太浅薄了...

FLOSS/自由开源软件, 当初是否可能被完全禁止?
不可能的:

+ 软件一开始就是开源的, 人类社会有种顽固性, 就是心理惯性, 无论什么事儿, 都将向方便滑落, 不可能越来越复杂难用, 所以, 原先代码是随时可以获得的, 到后来无论给出什么理由来, 照样有部分人, 单以方便为由就可能将自己代码公开, 不用任何许可证, 丢到公开域, Why not?
+ 自由/开源软件, 更加重要的体验就是 自由, 那种自在交流的感觉简直就是快乐本身, 这是用屎山代码换来的高工资无法替代的乐趣, 即便是没有 GPL , 人们也一样将自发形成小圈子来自由交流代码/项目/思想/作品/...这在其它行业也都有这种现象, 历史上也是, 只不过, 没有互联网形成自动传播效应...

快乐值多少銭? 这才是真正不想/无法评估的价值...

**Reply by lijiangsheng1** [reply]
从市场和企业的角度来看，开源只是实现他们商业的一部分，我们需要跳出来尝试用MBA之类的视角来看待，所以需要量化和核算。 他们根本不关心我们的快乐与否。

**Reply by ZoomQuiet** [reply]
#是也乎,(￣▽￣)
快乐无论是否关注总是在那儿, 只增不减;
理解无法是否努力总是在那儿, 等待创造;
是否快乐, 是我们自己的体验,
是否理解, 是对方素质...
